src/components/store/ProductDetails.tsx

```diff
@@ -142,10 +135,10 @@ export default function ProductDetails({
                     </div>
 
                     {/* Product Description */}
-                    <Product.Description>
-                      {({ plainDescription }) => (
+                    <ProductDescription as="html" asChild>
```

Comment:
see https://wix.slack.com/archives/C08TQK39HPG/p1755688374326999
should be just 
`<ProductDescription as="html" asChild>`
And the classNames in the p element should be in the shadcn component